### ISSUE TITLE
when will wav2vec 2.0 pretraining converge?

### ISSUE THREAD

**Issue by mcggood** [pytorch/fairseq#2981]
### Before asking:   
1. search the issues.   
2. search the docs.    

#### What is your question?
I'm running wav2vec 2.0 base pretraining with mandarin(200 hours). single tesla V100 32G batch_size 64.
And the log prints:
{"epoch": 10, "valid_loss": "3.801", "valid_ntokens": "1412.17", "valid_nsentences": "23.6442", "valid_prob_perplexity": "17.892", "valid_code_perplexity": "17.822", "valid_temp": "1.989", "valid_loss_0": "3.629", "valid_loss_1": "0.14", "valid_loss_2": "0.032", "valid_accuracy": "0.49559", "valid_wps": "23508", "valid_wpb": "1412.2", "valid_bsz": "23.6", "valid_num_updates": "1064", "valid_best_loss": "3.801"}
I noticed in the paper they trained the model for 1.5 days with 64 GPUs.
For I only have 8 GPUs, I want to know when is the pretraining model converge? what's the _**loss, valid_prob_perplexity, valid_code_perplexity and valid_accuracy**_ for Facebook libri-speech models?

#### What's your environment?

 - fairseq Version (e.g., 1.0 or master):
 - PyTorch Version (e.g., 1.0)
 - OS (e.g., Linux):
 - How you installed fairseq (`pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by mcggood** [comment]
It seems overfit just less than 20 epochs.
{"epoch": 18, "valid_loss": "3.242", "valid_ntokens": "1406.43", "valid_nsentences": "23.6442", "valid_prob_perplexity": "19.688", "valid_code_perplexity": "19.68", "valid_temp": "1.981", "valid_loss_0": "3.086", "valid_loss_1": "0.14", "valid_loss_2": "0.017", "valid_accuracy": "0.55915", "valid_wps": "23406.9", "valid_wpb": "1406.4", "valid_bsz": "23.6", "valid_num_updates": "1917", "valid_best_loss": "3.157"}

the best perplexity is epoch12, logs:
{"epoch": 12, "valid_loss": "3.455", "valid_ntokens": "1401.84", "valid_nsentences": "23.6442", "valid_prob_perplexity": "16.855", "valid_code_perplexity": "16.831", "valid_temp": "1.987", "valid_loss_0": "3.289", "valid_loss_1": "0.14", "valid_loss_2": "0.026", "valid_accuracy": "0.53597", "valid_wps": "23391.6", "valid_wpb": "1401.8", "valid_bsz": "23.6", "valid_num_updates": "1277", "valid_best_loss": "3.455"}

is that normal?

**Comment by edosyhptra** [comment]
Mine got loss_0: 2.246 and accuracy: 0.55 in epoch 23. I dont know it seems overfit or no. I want to ask aswell. Is mine normal?

[2021-06-22 14:14:45,462][fairseq.trainer][INFO] - NOTE: overflow detected, setting loss scale to: 0.03125
[2021-06-22 14:16:18,056][train_inner][INFO] - {"epoch": 23, "update": 0.864, "loss": "2.351", "ntokens": "23709.4", "nsentences": "145.23", "prob_perplexity": "279.599", "code_perplexity": "226.953", "temp": "1.987", "loss_0": "2.259", "loss_1": "0.081", "loss_2": "0.011", "accuracy": "0.54756", "wps": "1752.3", "ups": "0.07", "wpb": "23709.4", "bsz": "145.2", "num_updates": "1400", "lr": "7e-05", "gnorm": "0.462", "loss_scale": "0.0312", "train_wall": "1333", "wall": "17797"}
[2021-06-22 14:38:42,077][train_inner][INFO] - {"epoch": 23, "update": 0.925, "loss": "2.337", "ntokens": "23686", "nsentences": "144.26", "prob_perplexity": "279.732", "code_perplexity": "227.234", "temp": "1.986", "loss_0": "2.246", "loss_1": "0.081", "loss_2": "0.011", "accuracy": "0.5496", "wps": "1762.3", "ups": "0.07", "wpb": "23686", "bsz": "144.3", "num_updates": "1500", "lr": "7.5e-05", "gnorm": "0.461", "loss_scale": "0.0312", "train_wall": "1326", "wall": "19141"}
[2021-06-22 15:01:11,834][train_inner][INFO] - {"epoch": 23, "update": 0.986, "loss": "2.337", "ntokens": "23682.8", "nsentences": "145.09", "prob_perplexity": "279.642", "code_perplexity": "227.304", "temp": "1.985", "loss_0": "2.245", "loss_1": "0.081", "loss_2": "0.011", "accuracy": "0.55017", "wps": "1754.6", "ups": "0.07", "wpb": "23682.8", "bsz": "145.1", "num_updates": "1600", "lr": "8e-05", "gnorm": "0.459", "loss_scale": "0.0312", "train_wall": "1331", "wall": "20490"}
